### PR TITLE
Prepare v1.2.0

### PR DESCRIPTION
**Pull request by glpatcern**
As discussed today at the OCM weekly call, we wish to tag OCM v1.2, with the spirit of consolidating the recent improvements and propose the main vendors, Nextcloud, ownCloud, and Seafile, to include the implementation of the new standard in their development plans.

This PR includes a changelog of what comes in the new release. The real changes are already all in the `develop` branch.

I'd like to ask @schiessle, @butonic, @killing to "officially" approve this PR, with a prospected release date of Thursday, 18th July 2024, i.e. in a month from now.

CC @michielbdejong @mickenordin @smesterheide @MahdiBaghbani

### CODE REVIEW

**Comment by glpatcern** [comment]
Given the revived ongoing discussions after the holiday period, I think it makes sense to postpone this at least until #83, #92, #93 and #94 are resolved.

File: README.md [hunk 1]
```diff
@@ -94,10 +94,13 @@ Since there is no way to guarantee that the sharee OCM provider will actually en
 
 ## Contributing
 
-The specification can be rendered as HTML documentation using [ReDoc](https://github.com/Redocly/redoc) and is available as follows:
+The specification has been rendered as HTML documentation using [ReDoc](https://github.com/Redocly/redoc), and is available as follows:
```
Comment (michielbdejong) [inline README.md]: It gets rendered on-the-fly from spec.yml when you access those URLs, so I think 'can be' is more accurate than 'has been'?

Comment (glpatcern) [reply README.md]: That's a fair point! (I thought about the previous tagged versions but even those ones are rendered on the fly indeed...)

**Comment by glpatcern** [comment]
This is now ready, with a target release date for next Wednesday, 20 Nov 2024, on the occasion of the [OCM Workshop](https://indico.cern.ch/event/1451374). We can technically merge the PR earlier so that @michielbdejong can submit the text to the IETF.

Pinging again @schiessle, @butonic, @killing for information. Please check the changelog for the most significant changes, which notably include the security-enhancing features contributed by you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.2.0] - 2024-11-20 - Michiel B. De Jong <michiel@pondersource.com>
```
Comment (michielbdejong) [inline CHANGELOG.md]:
```suggestion
## [1.2.0] - 2024-11-20 - Michiel B. de Jong <michiel@pondersource.com>
```

**Review by michielbdejong — APPROVED** [review]
Congrats everyone on this milestone for Open Cloud Mesh! Look forward to discuss it with everyone at [the workshop on Wednesday](https://indico.cern.ch/event/1451374/)

**Pull request merged**